packages/driver/src/cy/commands/aliasing.ts

```diff
@@ -36,7 +35,7 @@ export default function (Commands, Cypress, cy) {
       const log: any = _.last(cy.queue.logs({
         instrument: 'command',
         event: false,
-        chainerId,
+        chainerId: this.get('chainerId'),
```

Comment:
No functional difference to this change, I just liked the way it looked better down here. :woman_shrugging: